src/components/dock/dock.scss

```diff
@@ -70,7 +70,7 @@ nav {
     flex-grow: 1;
 
```

Comment:
First of all, congrats and great job on your first PR! It's so exciting every time, even for me! 🤩